packages/core/contracts/financial-templates/long-short-pair/LongShortPairCreator.sol

```diff
@@ -128,10 +128,6 @@ contract LongShortPairCreator is Testable, Lockable {
         return lspAddress;
     }
 
-    /****************************************
```

Comment:
this comment was unnecessary. there are no other block comments in this contract